### PR TITLE
VIH-8537 display participant count in rooms

### PR DESCRIPTION
**Pull request by shaed-parkar**
### Jira link (if applicable)

VIH-8537

### Change description ###

* Refactor participant panel into item component
* Display a list of participants who are:
  * In hearing room
  * In waiting room
  * Not connected
* Display the total count of participants in the waiting room, which scrolls down to the section in the list when clicked

VIH-8537_screenshot

> NOTE: the participant list was duplicated as a quick hack to show case a long list. we do not have multiple judges
https://github.com/hmcts/vh-video-web/assets/41630528/d4ff2ae6-044e-4be7-81d8-7e8bf8f911f5

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html [hunk 1]
```diff
@@ -1,276 +1,124 @@
...
+      (click)="scrollToElement('panelListNotConnectedHeader')"
+    >
+      {{ 'participants-panel.in-waiting-room' | translate }} ({{ totalParticipantsInWaitingRoom }})
+    </a>
+  </div>
 
   <div class="button-row">
-    <div
+    <button
+      type="submit"
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html:17]: do these buttons still work? You've changed them to a submit, but there's no form

**Review by oliver-scott — COMMENTED** [review]
looks good, just left a comment about the buttons

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html (hunk 1 above)
Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html:17]: yup, I should probable make them normal button types though

Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html]: oh my bad I've been testing this with the keyboard only! great catch Oliver!

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**